firebase-vscode/src/logger-wrapper.ts

```diff
@@ -45,7 +46,11 @@ export function logSetup() {
     // Re-implement file logger call from ../../src/bin/firebase.ts to not bring
     // in the entire firebase.ts file
     const rootFolders = getRootFolders();
-    const filePath = path.join(rootFolders[0], ".firebase", "logs", "vsce-debug.log");
+    // Default to a central path, but write files to a local path if we're in a Firebase directory.
+    let filePath = "~/.cache/firebase/vscode/vsce-debug.log";
```

Comment:
```suggestion
    let filePath = "~/.cache/firebase/logs/vsce-debug.log";
```

And we also need a random segment to prevent conflicts across multiple windows.

And speaking of windows... Does it work on Windows (capital W)?

Reply:
And... one last thing, please use `require('os').homedir()` instead of `~` for portability

Reply:
Good calls, done. Ignoring the random segment bit for now - different instances will write to the same log, which is fine for now imo.